cypress/support/commands.js

```diff
@@ -47,24 +47,35 @@ Cypress.Commands.add('login', (email, username, password) => {
   });
 });
 
-Cypress.Commands.add('createArticle', (title, description, body) => {
-  cy.getCookie('auth').then((token) => {
-    const authToken = token.value;
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
+  cy.get(`[placeholder="${placeholder}"]`);
```

Comment:
The `findByPlaceholder` command should return the result of `cy.get` to allow chaining with other Cypress commands. Consider adding `return` before `cy.get`.